FirstDeliverable/CSV/csv.go

```diff
@@ -0,0 +1,67 @@
+package csv
+
+import (
+	"encoding/csv"
+	"encoding/json"
+	"fmt"
+	"io"
+	"log"
+	"os"
+)
+
+type Item struct {
+	ItemID   string `json:"item_id"`
```

Comment:
These fields belong to the Item struct, so put Item at the beginning is redundant:
```go 
// Item - some description here
type Item struct {
    ID       string `json:"item_id"`
    Name string `json:"item_name"`
    Type  string `json:"item_type"`
```